addons/sale_purchase_stock/views/purchase_order_views.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="purchase_order_form_sale_purchase_stock" model="ir.ui.view">
+            <field name="name">purchase.order.form.sale.purchase.stock</field>
+            <field name="model">purchase.order</field>
+            <field name="inherit_id" ref="purchase.purchase_order_form"/>
```

Comment:
`purchase_stock.purchase_order_view_form_inherit`

so that you ensure you add it to the view when `dest_address_id` is already added to the view